File: bitSetGo-v2.0/D-Gamealarm/logic.cpp
#define ll long long int
#include<bits/stdc++.h>
using namespace std;
int main()
{
ll tc;
cin>>tc;
while(tc--)
{
ll n,k;
cin>>k>>n;
ll ans=n/9;
ll rem=n%9;
if(k%9==0)
k=0;
for(int i=n-rem+1;i<=n;i++)
{
if(i%9==k)
{
ans++;
break;
}
}
cout<<ans<<endl;
}
return 0;
}

File: bitSetGo-v2.0/D-Gamealarm/logic.java
import java.util.*;
public class abc
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        long  tc=sc.nextLong();
        for(long  i=0;i<tc;i++)
        {
            long  k=sc.nextLong();
            long  n=sc.nextLong();
            long  ans=n/9;
            long  rem=n%9;
            if(k%9==0)
            k=0;
           // System.out.print(" "+ans);
            for(long  j=n-rem+1;j<=n;j++)
            {
                if(j%9==k)
               { ans++;
                break;}
            }
        
            System.out.println(ans);
        }
        

    }
}